dGame/dComponents/RocketLaunchLupComponent.cpp

```diff
@@ -1,21 +1,17 @@
 #include "RocketLaunchLupComponent.h"
-#include "CDClientDatabase.h"
 #include "RocketLaunchpadControlComponent.h"
 #include "InventoryComponent.h"
 #include "CharacterComponent.h"
 
 RocketLaunchLupComponent::RocketLaunchLupComponent(Entity* parent) : Component(parent) {
 	m_Parent = parent;
-
-	// get the lup worlds from the cdclient
-	std::string query = "SELECT * FROM LUPZoneIDs;";
-	auto results = CDClientDatabase::ExecuteQuery(query);
-	while (!results.eof()) {
-		// fallback to 1600 incase there is an issue
-		m_LUPWorlds.push_back(results.getIntField(0, 1600));
-		results.nextRow();
+	std::string zoneString = GeneralUtils::UTF16ToWTF8(m_Parent->GetVar<std::u16string>(u"MultiZoneIDs"));
+	std::stringstream ss(zoneString);
```

Comment:
Believe this is unnecessary.  There should be a tool in general utils to split a string by a delimiter ans return a vector of the values.

Reply:
There are but none return the right type